nixos/doc/manual/development/replace-modules.xml

```diff
@@ -0,0 +1,36 @@
+<section xmlns="http://docbook.org/ns/docbook"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        version="5.0"
+        xml:id="sec-replace-modules">
+
+<title>Replace Modules</title>
+
+<para>It's also possible to disable modules allowing the
+  implementation to be replaced.  This can be used to import a set of
```

Comment:
Can you clarify what does it mean. Is that supposed to replace only all option definitions?

Note that `implementation` is still used in many modules as a comment above the `config` attribute, as the `interface` is used above the `options` attributes.  The proper names are `options definitions` and `option declarations`, respectively for `config` and `options` attribute sets.